diff -ru clearly/cmd/test clearly-lib/cmd/test
--- clearly/cmd/test	2020-04-07 12:19:37.054609706 -0600
+++ clearly-lib/cmd/test	2020-04-15 16:36:55.128831767 -0600
@@ -662,7 +662,7 @@
     TEST_INSTALLED=yes
     TEST_GITWD=
     TEST_DIR=${clearly_base}/libexec/clearly/test
-    TEST_EXAMPLES_DIR=${clearly_base}/share/doc/clearly/examples
+    TEST_EXAMPLES_DIR=${clearly_base}/share/doc/clearly-${clearly_version}/examples
 else
     # build dir
     TEST_INSTALLED=
